module/utils.nf

```diff
@@ -19,18 +49,13 @@ process compress_and_index_HTSlib {
 
     script:
     """
-    bgzip ${tsv} --output output.tsv.gz
-
-    tabix \
-        --sequence 1 \
-        --begin 2 \
-        --end 2 \
-        output.tsv.gz
+    bgzip ${input_tsv} --output output.tsv.gz
+    tabix -s 1 -b 2 -e 2 output.tsv.gz
     """
 
     stub:
     """
-    touch "output.tsv.gz"
-    touch "output.tsv.gz.tbi"
+    touch output.tsv.gz
+    touch output.tsv.gz.tbi
     """
-}
+}
```

Comment:
**quibble (non-blocking):** I will _happily_ die on this hill: [files must end with newlines](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)!

Code editors should do this for you automatically - whatever tool last touched this file isn't doing the right thing.

Reply:
That tool was *me*, does your vscode automatically add new lines?

Reply:
It seems like it's a whole _thing_ with VSCode and extensions:

https://rfdonnelly.github.io/posts/sane-vscode-whitespace-settings/

Reply:
Also:

![image](https://github.com/user-attachments/assets/716936ce-edf3-410c-9ded-efa1132a6111)

Reply:
![image](https://github.com/user-attachments/assets/06e04eda-4586-47f0-a289-9225ad4295a2)